src/vs/base/browser/ui/selectBox/selectBoxNative.ts

```diff
@@ -35,6 +35,10 @@ export class SelectBoxNative extends Disposable implements ISelectBoxDelegate {
 			this.selectElement.setAttribute('aria-label', this.selectBoxOptions.ariaLabel);
 		}
 
+		if (this.selectBoxOptions.ariaDescription) {
```

Comment:
Same here - `typeof this.selectBoxOptions.ariaDescription === 'string'`

Reply:
done